pg-extend/Cargo.toml

```diff
@@ -26,4 +26,5 @@ postgres-11 = []
 [dependencies]
 
 [build-dependencies]
-bindgen = "0.46"
+bindgen = "0.41"
```

Comment:
bindgen 0.42+ requires clang-sys 0.26. clang requires clang-sys 0.23. This downgrade doesn't appear to break anything and solves this dep conflict.

Reply:
...https://github.com/rust-lang/rust-bindgen/pull/1410 ah maybe this isn't an option. hmm.